modoptions.lua

```diff
@@ -2144,6 +2114,16 @@ Example: Armada VS Cortex VS Legion: 273 or 100 010 001 or 256 + 16 + 1]],
 		def		= false,
 	},
 
+    {
+		key		= "holiday_events",
```

Comment:
```suggestion
		key		= "disable_holiday_events",
```
to avoid inverting the meaning of the key